Config.mk

```diff
@@ -60,6 +60,13 @@ ifeq ($(OS), DragonFly)
 	LDFLAGS  += -L/usr/local/lib
 endif
 
+# Which search priority to use for libraries (system or local)
+CUSTOM_LIB_SEARCH_PRIORITY := local
```

Comment:
This is a way to prioritize the local over the system libs as most developers will need to build grpc, protobuf, and absl from scratch if there is an incompatibility issue. Check the versions inside for what versions you need to clone: https://github.com/k0T0z/absl-proto-grpc-ci